sale_order_general_discount_triple/models/sale_order_line.py

```diff
@@ -4,51 +4,81 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    discount = fields.Float(
-        string="Discount (%)",
-        digits="Discount",
-        compute=False,
-        store=True,
-        readonly=False,
-        default=0.0,
-    )
-
-    @api.model
-    def default_get(self, fields):
-        vals = super().default_get(fields)
-        general_discount = (
+    discount2 = fields.Float(compute="_compute_discount2", store=True, readonly=False)
+    discount3 = fields.Float(compute="_compute_discount3", store=True, readonly=False)
```

Comment:
these need to be `precompute=True` to avoid these warnings
 `Field sale.order.line.price_reduce_taxinc cannot be precomputed as it depends on non-precomputed field sale.order.line.price_total`
`Field sale.order.line.price_total cannot be precomputed as it depends on non-precomputed field sale.order.line.discount2`
+ a few more caused by the same root
I will push the change